src/app/joint-control/joint-control-core/joint-control-core.component.html

```diff
@@ -11,42 +11,33 @@
             <div class="col-3 sidebar-label m-0 p-0">Active</div>
             <div class="col-3 sidebar-label m-0 p-0">Current</div>
         </div>
-        <!--
-            In ngClass you can find the following statement: [motor.motorName == motorName && motor.motorName != 'all_fingers_left' && motor.sourceMotorName != 'all_fingers_right' || ( (motorName == 'all_fingers_left' || motorName == 'all_fingers_right') && motor.motorName != 'all_fingers_left' && motor.sourceMotorName != 'all_fingers_right') ? 'backgroundSelected' : ''  
-            This is used to higlite all rows when a motor is selected. This excludes the row of all_fingers_left and all_fingers_right
-        -->
         <div
-            *ngFor="let motor of joint.motors"
+            *ngFor="let motor of displayMotors"
             class="row m-0 p-0"
-            [ngClass]="[motor.motorName == motorName && motor.motorName != 'all_fingers_left' && motor.motorName != 'all_fingers_right' || ( (motorName == 'all_fingers_left' || motorName == 'all_fingers_right') && motor.motorName != 'all_fingers_left' && motor.motorName != 'all_fingers_right') ? 'backgroundSelected' : '',
-                joint.reversed ? 'flex-row-reverse' : 'flex-row']"
+            [ngClass]="[joint.reversed ? 'flex-row-reverse' : 'flex-row', motor.motorName == selectedMotor?.motorName ? 'background-selected' : '']"
```

Comment:
we still need to check here for `selectedMotor.displaySettings` (i.e. is an all-fingers-motor selected?). Also, `motor.motorName == selectedMotor?.motorName` can be simplified to `motor === selectedMotor`.